Documentation/requirements.txt

```
@@ -1,6 +1,6 @@
 linkify-it-py
 myst-parser
-sphinx
+sphinx == 6.1.3
```

```suggestion
sphinx ~= 6.1.3
```

See https://peps.python.org/pep-0440/#compatible-release

---

Did you happen to also try `sphinx-notfound-page==1.0.0rc1` as suggested in https://github.com/readthedocs/sphinx-notfound-page/issues/219#issuecomment-1691765304 ?

---

I just double-checked that and it does seem to be working. I think we should move forward with setting the version of the `sphinx-notfound-page` package